Podcast/BookmarkViewController.swift

```diff
@@ -15,6 +15,7 @@ class BookmarkViewController: UIViewController, UITableViewDelegate, UITableView
     ///
     var bookmarkTableView: UITableView!
     var cards: [Card] = []
+    var currentlyPlayingIndexPath: IndexPath?
```

Comment:
I think this might complicate things, since I need to call `setPlayButtonToState(isPlaying: false)` on the previously playing cell, which i get through calling `bookmarkTableView.cellForRow(at: indexPath)`. If we change it to an Int, I don't believe there's a function to get a cell from just an Int